### PR TITLE
LPS-159271 Add the email address of the impersonated user

### PR DESCRIPTION
**Pull request by nikolettabuza**
[LPS-159271](https://liferay.atlassian.net/browse/LPS-159271)

### CODE REVIEW

File: portal-kernel/src/com/liferay/portal/kernel/audit/AuditMessage.java [hunk 1]
```diff
@@ -100,6 +103,24 @@ public AuditMessage(
...
+
+			_additionalInfoJSONObject.put(
+				"doAsUserEmailAddress", PortalUtil.getUserEmailAddress(doAsUserId)
+			).put(
+				"doAsUserId", String.valueOf(doAsUserId)
+			).put(
+				"doAsUserName", PortalUtil.getUserName(doAsUserId, StringPool.BLANK)
+			);
+		}
+
```
Comment (stian-sigvartsen) [inline portal-kernel/src/com/liferay/portal/kernel/audit/AuditMessage.java:123]: Very nice @nikolettabuza !

**Comment by nikolettabuza** [comment]
ci:test:sf

**Comment by nikolettabuza** [comment]
ci:test:sf

**Comment by nikolettabuza** [comment]
ci:test:relevant

**Comment by stian-sigvartsen** [comment]
The CI failures are unrelated.

**Comment by stian-sigvartsen** [comment]
Good job @nikolettabuza ! There seems to be some duplication in the changes but it is not doing any harm so I am going to forward this. We can follow up in the morning.

**Comment by stian-sigvartsen** [comment]
ci:forward:force

**Pull request closed without merging**